[sr_movements] image magick warnings

```
In file included from /usr/include/ImageMagick-6/Magick++/Include.h:14:0,
                 from /usr/include/ImageMagick-6/Magick++.h:10,
                 from /catkin_melodic_robots/src/sr_tools/sr_movements/include/sr_movements/movement_from_image.hpp:32,
                 from /catkin_melodic_robots/src/sr_tools/sr_movements/src/movement_from_image.cpp:30:
/usr/include/ImageMagick-6/magick/magick-config.h:29:3: warning: #warning "you should set MAGICKCORE_QUANTUM_DEPTH to sensible default set it to configure time default" [-Wcpp]
 # warning "you should set MAGICKCORE_QUANTUM_DEPTH to sensible default set it to configure time default"
   ^~~~~~~
/usr/include/ImageMagick-6/magick/magick-config.h:30:3: warning: #warning "this is an obsolete behavior please fix your makefile" [-Wcpp]
 # warning "this is an obsolete behavior please fix your makefile"
   ^~~~~~~
/usr/include/ImageMagick-6/magick/magick-config.h:52:3: warning: #warning "you should set MAGICKCORE_HDRI_ENABLE to sensible default set it to configure time default" [-Wcpp]
 # warning "you should set MAGICKCORE_HDRI_ENABLE to sensible default set it to configure time default"
   ^~~~~~~
/usr/include/ImageMagick-6/magick/magick-config.h:53:3: warning: #warning "this is an obsolete behavior please fix yours makefile" [-Wcpp]
 # warning "this is an obsolete behavior please fix yours makefile"
```

could you please create a melodic branch there too, then I will provide a PR

--- a/sr_movements/CMakeLists.txt
+++ b/sr_movements/CMakeLists.txt
@@ -11,6 +11,9 @@ catkin_package(
     LIBRARIES sr_movement
 )
 
+add_definitions(-DMAGICKCORE_QUANTUM_DEPTH=16)
+add_definitions(-DMAGICKCORE_HDRI_ENABLE=0)
+
 add_executable(sr_movements src/sr_movements.cpp src/movement_from_image.cpp src/partial_movement.cpp src/movement_publisher.cpp)
 add_dependencies(sr_movements ${catkin_EXPORTED_TARGETS})
 target_link_libraries(sr_movements ${ImageMagick_LIBRARIES} ${catkin_LIBRARIES})

